### PR TITLE
populate db with data throught the API

### PR DESCRIPTION
**Pull request by isabelrios**
So that we can run the OGC tests we would need data into the DB before.

### CODE REVIEW

**Comment by isabelrios** [comment]
@ferjm please when you have time take a look at this script to populate the db and let me know your thinking.

Thanks!

**Comment by isabelrios** [comment]
Thanks for your comments! tried to fix them, hope that looks better now. @ferjm r?

File: scripts/populateDB.sh [hunk 1]
```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env bash
+
+export HOST=http://localhost:8080
```
Comment (ferjm) [inline scripts/populateDB.sh]: Oops... this needs to be `http://localhost:8080/v1.0` after https://github.com/mozilla-sensorweb/sensorthings/commit/169076952faa929aaad1ce8509cf146ddac50904 landed

**Comment by ferjm** [comment]
Thank you!

File: scripts/populateDB.sh (hunk 1 above)
Comment (isabelrios) [inline scripts/populateDB.sh]: Perfect example of why it should be in a variable :) Changed!

**Pull request merged**